kedro/framework/cli/project.py

```diff
@@ -53,6 +53,8 @@
 override the loaded ones."""
 PIPELINE_ARG_HELP = """Name of the registered pipeline to run.
 If not set, the '__default__' pipeline is run."""
+NAMESPACE_ARG_HELP = """Name of the namespace to run.
+If not set, no namespace is used."""
```

Comment:
```suggestion
NAMESPACE_ARG_HELP = """Name of the node namespace to run."""
```
IMO the "no namespace is used" just makes this less clear since it can't be really explained fully in just a line.